administrator/language/de-DE/plg_multifactorauth_yubikey.sys.ini

```
@@ -0,0 +1,8 @@
+; Joomla! German Translation
+; (C) 2022 Open Source Matters, Inc. <https://www.joomla.org>
+; (C) Translation 2008 - 2022 J!German <https://www.jgerman.de>
+; License GNU General Public License version 2 or later; see LICENSE.txt
+; Note : All ini files need to be saved as UTF-8
+
+PLG_MULTIFACTORAUTH_YUBIKEY="Multi-Faktor-Authentifizierung - YubiKey"
+PLG_MULTIFACTORAUTH_YUBIKEY_XML_DESCRIPTION="Ermöglicht Benutzern auf der Website die Verwendung der Multi-Faktor-Authentifizierung mit einem sicheren YubiKey-Hardware-Token. Benutzer benötigen ihren eigenen YubiKey, der unter https://www.yubico.com/ erhältlich ist. Um die Multi-Faktor-Authentifizierung zu verwenden, müssen Benutzer ihr Benutzerprofil bearbeiten und die Multi-Faktor-Authentifizierung aktivieren."
```

```suggestion
PLG_MULTIFACTORAUTH_YUBIKEY_XML_DESCRIPTION="Ermöglicht Benutzern auf der Website die Verwendung der Multi-Faktor-Authentifizierung mit einem sicheren YubiKey-Hardware-Token. Benutzer benötigen ihren eigenen YubiKey, der unter https://www.yubico.com/ erhältlich ist. Um die Multi-Faktor-Authentifizierung zu verwenden, müssen Benutzer ihr Benutzerprofil bearbeiten und die Multi-Faktor-Authentifizierung aktivieren."
```
auch hier verlinken und rel noopener und noreferer